src/ResourceManager/DataLakeStore/Commands.DataLakeStore/help/Get-AzureRmDataLakeStoreChildItemProperties.md

```diff
@@ -0,0 +1,260 @@
+```
+
+### GetAllProperties
+```
+Get-AzureRmDataLakeStoreChildItemProperties [-Account] <String> [-Path] <DataLakeStorePathInstance>
+ [-OutputPath] <String> [-SaveToAdl] [-IncludeFiles] [-MaximumDepth <Int32>] [-Concurrency <Int32>]
+ [-GetDiskUsage] [-GetAcl] [-HideConsistentAcls] [-DefaultProfile <IAzureContextContainer>] [-WhatIf]
+ [-Confirm] [<CommonParameters>]
+```
+
+### GetAclDump
+```
+Get-AzureRmDataLakeStoreChildItemProperties [-Account] <String> [-Path] <DataLakeStorePathInstance>
+ [-OutputPath] <String> [-SaveToAdl] [-IncludeFiles] [-MaximumDepth <Int32>] [-Concurrency <Int32>] [-GetAcl]
+ [-HideConsistentAcls] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+{{Fill in the Description}}
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> {{ Add example code here }}
+```
+
+{{ Add example description here }}
+
+## PARAMETERS
+
+### -Account
+The Data Lake Store account to execute the filesystem operation in
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases: AccountName
+
+Required: True
+Position: 0
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -Concurrency
+Indicates the number of files/directories processed in parallel.
+Default will be computed as a best effort based on system specification.
+
+```yaml
+Type: Int32
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with Azure.
+
+```yaml
+Type: IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -GetAcl
+Retrieves the acl starting from the root path
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: GetAllProperties, GetAclDump
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -GetDiskUsage
+Retrieves the disk usage starting from the root path
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: GetDiskUsage, GetAllProperties
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -HideConsistentAcls
+Do not show directory subtree if the ACLs are the same throughout the entire subtree. This makes it easier to see only the paths up to which the ACLs differ.For example if all files and folders under /a/b are the same, do not show the subtreeunder /a/b, and just output /a/b with 'True' in the Consistent ACL columnCannot be set if IncludeFiles is not set, because consistent Acl cannot be determined without retrieving acls for the files.```yaml
```

Comment:
@rahuldutta90 please move the trailing ` ```yaml ` string to a line by itself (see the above parameters for reference)

Reply:
@cormacpayne Fixed